### DISCUSSION TITLE
[Ozone 1.3.0] OM how renew expired certificate?

### DISCUSSION THREAD

**Discussion by yazynin** [apache/ozone#6687] [FAQ] [answered]
How I can renew OM expired certificate?, work around from Cloudera not helped
`
Stop all the Ozone roles except the Storage Container Managers (SCM)
For every host with an Ozone role added, locate the directories that are storing Ozone internal certificates (Since SCM certificates expire in 5 years, they can be untouched): find / -name ozone-metadata 2>/dev/null | grep -v "scm" | while read line; do find $line -name certs; done
Note
The S3G Gateway role does not have SCM-generated certificates. You can ignore the steps on the S3 gateway role.
Move these certs directories to a backup location
For every host with an Ozone role added, locate the directories that are storing key material for Ozone internal certificates (do not touch SCMs): find / -name ozone-metadata 2>/dev/null | grep -v "scm" | while read line; do find $line -name keys; done
Note
The S3G Gateway role does not have SCM-generated certificates. You can ignore the steps on the S3 gateway role.
Move these keys directories to a backup location
For every host with an Ozone Manager role, locate the VERSION file of the Ozone Manager: find / -name om 2>/dev/null | while read line; do find $line -name VERSION; done | sort | uniq
Backup the version file (just in case you need to restore for any reason)
In the version file, remove the line starting with omCertSerialId in the file
Start the stopped Ozone roles and certificates will be regenerated during startup
`
Om not started with error: "2024-05-16 15:52:46,482 ERROR om.OzoneManagerStarter: OM start failed with exception
java.lang.RuntimeException: OzoneManager started in secure mode but doesn't have SCM signed certificate."

**Comment by ChenSammi** [comment, marked as answer]
Ozone 1.3.0 is a quite old version. I would suggest to upgrade to 1.4.0 if possible. 
For this issue, after remove the certs and keys directory, and remove omCertSerialId from OM version file, kindly try re-init OM, command like this,
"ozone om --init"

**Reply by yazynin** [reply]
thank you
